bin/chandra_repro

```diff
@@ -137,6 +137,66 @@ def error_out(params,msg,val):
     print("\n%s" % msg)
     raise ValueError("%s" % val)
 
+
+def filter_aspsolobi_from_aspsol(asol_list,mydir):
+    """As of DS10.8.3, there are now two flavors of aspect solution 
+    files, per OBI (CONTENT=ASPSOLOBI) and the original per-constant
+    aspect interval (cai) (CONTENT=ASPSOL).  Users should only ever
+    have one or the other. 
+    
+    Unfortunately, due to a quirk in the archive, users may have both.
+    
+    We know that there can be 1 and only 1 ASPSOLOBI file per obi (multi-obi
+    obsid's will need to have already been split, #repro5). 
+    
+    So if we find both ASPSOLOBI and ASPSOL files, we are going to pick
+    the ASPSOLOBI files.  This provides a better off-axis angle value
+    and tighter FOV.
+    """
+    from pycrates import read_file
+    from collections import defaultdict
+    
+    content_values = defaultdict(list)
+    for f in asol_list:
+        tab = read_file(f.strip(), mode="r")
+        content = tab.get_key_value("CONTENT")
+        content_values[content].append(f)
+        
+    if "ASPSOLOBI" in content_values:
+        if len( content_values["ASPSOLOBI"] ) > 1:
+            error_out("Too many ASPSOLOBI aspect file found.  If this "+
+            "is a mutli-obi observations, be sure to split it first "+
+            "using the splitobs script")
+
+    if "ASPSOL" in content_values and "ASPSOLOBI" in content_values:
+        v2("WARNING: The primary directory has both CONTENT=ASPSOL and "+
+          "CONTENT=ASPSOLOBI files. Only the ASPSOLOBI file "+
+          "{} will be used".format(content_values["ASPSOLOBI"]))
+
+        # Wwant to go ahead and remove the old files from the
+        # output directory.
+        for oldasp in content_values["ASPSOL"]:
+            old = os.path.join(mydir, os.path.basename(oldasp.strip()))
+            if os.path.exists(old):
+                os.unlink(old)
+
+        # may not be necessary to del but I don't like relying on
```

Comment:
An alternative would be to set up booleans earlier, like

```python
has_aspsolobi = 'ASPSOLOBI' in content_values
has_aspsol = 'ASPSOL' in concent_values
```

and then use these in the checks.